app/helpers/admin_helper.rb

```diff
@@ -49,11 +49,15 @@ def format_address(*parts)
     safe_join(parts.compact_blank, tag.br)
   end
 
-  def admin_participant_header_and_title(profile:, section:)
+  def admin_participant_header_and_title(presenter:, section:)
+    profile = presenter.participant_profile
     full_name = profile.full_name
     role = admin_participant_role_name(profile.class.name)
     trn = profile.teacher_profile.trn
-    cohort = profile.cohort&.start_year
+
+    # TODO: There should be a single consistent way to find the relevant cohort location
```

Comment:
There really should.

Reply:
moving it to the presenter at least is a start I guess